### DISCUSSION TITLE
Simple model rendering

### DISCUSSION THREAD

**Discussion by TopperDEL** [StereoKit/StereoKit#113] [Q&A] [answered]
I have the attached Model but can get it to render. I'm loading it with Model.FromFile("BuckyBoall.stl") and draw it a a location, that is working with Mesh.GenerateSphere().

What could be wrong here?
[BuckyBoall.zip](https://github.com/maluoi/StereoKit/files/5932872/BuckyBoall.zip)

**Comment by maluoi** [comment, marked as answer]
Whups, looks like I never got around to binary format .stl files! 😬 Current implementation is ASCII .stl only! This can be tracked as #114

**Reply by TopperDEL** [reply]
I've converted the file to a GLB and it didn't work either. Do currently all Model-Files need to be ASCII-based?

**Reply by TopperDEL** [reply]
And an ASCII-based STL is also not working (see attached file).
[Ball.zip](https://github.com/maluoi/StereoKit/files/5934776/Ball.zip)

**Reply by TopperDEL** [reply]
Found it. That object is simply so huge that I'm always completely in it. Beginner's error, right? :)
So I simply had to scale it down accordingly.

**Reply by maluoi** [reply]
GLB should be fine! I have plans to eventually do full conformance testing on the GLTF format, as I consider it SK's native format. I have put it through some paces already, so it should generally be reliable.

Scale is a bit of a wild west! GLTF at least is strictly defined as meters, but it's still tricky to get that right in modeling tools. First thing I always do when modeling is add a human sized box for reference :D

**Reply by TopperDEL** [reply]
I also tried a glb and it showed nothing. But I think that has been the same issue with scaling. So I'm fine now with this topic here. 😊
